lib/git-lfs-s3/services/ceph_presigner.rb

```diff
@@ -0,0 +1,24 @@
+require 'base64'
+require 'date'
+require 'digest/sha1'
+require 'openssl'
+require 'uri'
+
+module GitLfsS3
+  module CephPresignerService
+    extend self
+    extend AwsHelpers
+
+    def signed_url(obj)
+      expire_at = (DateTime.now + 1).strftime("%s")
+      secret_access_key = Aws.config[:secret_access_key]
+      access_key_id = Aws.config[:access_key_id]
```

Comment:
Unlike aws-sdk v1, in aws-sdk v2 Aws.config is simply a vanilla ruby hash. See:

http://docs.aws.amazon.com/sdkforruby/api/index.html

But there is definitely magic going on there that is external to this library. See: https://github.com/lsst-sqre/git-lfs-s3-server/blob/master/git-lfs-s3-server.rb#L16-L23
So I'll move it back to GitLfsS3::Application.settings.